eclipse-theia/theia
# vscode: add `hcLight` support

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/11314.
The pull-request adds support for **high contrast light themes** and includes the following updates:
- adds support for `hcLight` themes
- updates our color registrations to include `hcLight` based on values present in vscode
- updates our `change color theme` command to support `hcLight` and adds localizations

![Screen Shot 2022-08-19 at 9 17 35 AM](https://user-images.githubusercontent.com/40359487/185626980-64086b0c-3ff2-437e-a8bf-467cdac3f7d0.png)


#### How to test


1. include the following test theme [theia-hclight-0.0.2.zip](https://github.com/eclipse-theia/theia/files/9992348/theia-hclight-0.0.2.zip)
2. start the application
3. execute the command `Preferences: Color Theme` - the high contrast light themes should be under `high contrast`, selecting `Test High Contrast Light Theme` should work
4. the new `High Contrast Light (Theme)` should also work

#### Review checklist


#### Reminder for reviewers


- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
